### DISCUSSION TITLE
Docker storage and logo

### DISCUSSION THREAD

**Discussion by cirrusflyer** [owncast/owncast#514] [Q&A] [answered]
Can you provide guidance on when using Docker, to use your own logo?

**Comment by geekgonecrazy** [comment, marked as answer]
Actually answered this for someone else on mastodon:

https://fosstodon.org/@geekgonecrazy/105420678558002246

add volume in your docker command something like this:

`-v ${PWD}/img/logo.png:/app/webroot/img/logo.png`

Will mount a local image into the container.  Then just make sure it’s in your config file to use logo.png
